### ISSUE TITLE
word wrap for longer text?

### ISSUE THREAD

**Issue by shiffman** [ITPNYU/ICMadness-2018#23]
Longer text does not fit, perhaps we can wrap when necessary?

![screen shot 2018-11-01 at 9 19 56 pm](https://user-images.githubusercontent.com/191758/47888641-f757a680-de1b-11e8-82b6-368f2cf7527d.png)

**Comment by mithru** [comment]
I'm wondering how to best do this. Wrapping it affects the animation since that uses the positions of the elements relative to each other.  :( 

I'm sure there's a different way of implementing the animation. I didn't consider this edge case when I wrote it.

**Comment by mithru** [comment]
maybe we could wrap only when the spinner stops?

**Comment by shiffman** [comment]
Yes wrapping just at the end would work fine I think! We could also just rewrite that one item. 😂

**Comment by mithru** [comment]
#27 should fix this, but the alignment isn't perfect on wider screens. close enough for tomorrow, maybe?

**Comment by shiffman** [comment]
Leaving this open in case we want to re-address later but we're good for today I think!
